From 14e561370a2c4194d26e0b7b016925256890c082 Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Wed, 18 Dec 2024 01:51:22 +0900
Subject: [PATCH 1/2] fix for armhf

---
 CMakeLists.txt | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b52a9bb298..247bf57e80 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,10 +24,23 @@ if($ENV{ROS_DISTRO} STREQUAL "indigo" OR $ENV{ROS_DISTRO} STREQUAL "kinetic" OR
     PYTHON_INTERPRETER python2
   )
 else()
+  execute_process(
+    COMMAND dpkg-architecture -qDEB_TARGET_ARCH
+    OUTPUT_VARIABLE DEB_TARGET_ARCH
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+  message(STATUS "Debian target architecture: ${DEB_TARGET_ARCH}")
+  if("${DEB_TARGET_ARCH}" STREQUAL "armhf")
+  catkin_generate_virtualenv(
+    INPUT_REQUIREMENTS requirements.in.armhf
+    PYTHON_INTERPRETER python3
+  )
+  else()
   catkin_generate_virtualenv(
     INPUT_REQUIREMENTS requirements.in
     PYTHON_INTERPRETER python3
   )
+  endif()
 endif()
 
 file(GLOB SCRIPTS_FILES scripts/*)
-- 
2.17.1

