From 375184c50286c0ae9dba1be2b5d80b98b88c166f Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Fri, 16 Sep 2016 17:17:05 +0900
Subject: [PATCH 2/3] python-texttable is not available

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index a65860d..8ee5ffe 100644
--- a/debian/control
+++ b/debian/control
@@ -8,5 +8,5 @@ Standards-Version: 3.9.2
 
 Package: ros-indigo-jsk-tools
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, iproute2, python-colorama, python-texttable, ros-indigo-axis-camera, ros-indigo-jsk-network-tools, ros-indigo-jsk-topic-tools, ros-indigo-pr2-computer-monitor, ros-indigo-rosbag, ros-indigo-rosemacs, ros-indigo-rosgraph-msgs, ros-indigo-rospy, ros-indigo-rqt-reconfigure
+Depends: ${shlibs:Depends}, ${misc:Depends}, iproute2, python-colorama, ros-indigo-axis-camera, ros-indigo-jsk-network-tools, ros-indigo-jsk-topic-tools, ros-indigo-pr2-computer-monitor, ros-indigo-rosbag, ros-indigo-rosemacs, ros-indigo-rosgraph-msgs, ros-indigo-rospy, ros-indigo-rqt-reconfigure
 Description: Includes emacs scripts, ros tool alias generator, and launch doc generator.
-- 
1.9.1

